.github/workflows/build-test-image.yaml

```
@@ -141,8 +161,8 @@ jobs:
           file: ${{ matrix.dockerfile }}
           push: true
           tags: falkordb/${{ matrix.image-name }}:dev-${{ github.event.head_commit.id || github.run_id }}
-          cache-from: type=local,src=/tmp/.buildx-cache
-          cache-to: type=local,dest=/tmp/.buildx-cache,mode=max
+          cache-from: type=gha,scope=${{ matrix.image.image_name }}
```

actionlint (1.7.12)

[error] property "image" is not defined in object type {dockerfile: string; image-name: string}

(expression)